denops/glance/main.ts

```diff
@@ -1,12 +1,27 @@
 import { Denops } from "https://lib.deno.dev/x/denops_std@v3/mod.ts";
-import { execute } from "https://lib.deno.dev/x/denops_std@v3/helper/mod.ts";
 import { g } from "https://lib.deno.dev/x/denops_std@v3/variable/mod.ts";
 import * as fn from "https://lib.deno.dev/x/denops_std@v3/function/mod.ts";
 import { join } from "https://lib.deno.dev/std/path/mod.ts";
 import { Server } from "./server.ts";
 import { MarkdownRenderer } from "./markdown.ts";
 
+type Options = {
+  port: number;
+  plugins: string[];
+  html: boolean;
+  breaks: boolean;
+  linkify: boolean;
+  defaultStylesheet: string;
+  stylesheet: string;
+  defaultConfigPath: string;
```

Comment:
`defaultStylesheet` and `defaultConfigPath` are default values. I mean that these are not configuration parameter of Glance.vim. Would you mind to remove them from code?

Reply:
Sure, gonna remove them.